### ISSUE TITLE
OOD: Document the workflow that will be used

### ISSUE THREAD

**Issue by f-PLT** [RolnickLab/antenna#748]
## Summary

Document the workflow/processes that are expected to be executed daily/weekly.

## Detailed Description

Detailed documentation of all the steps and their requirements. Ex. Upload images, clustering, etc.

## Related Issues

#747

**Comment by mihow** [comment]
@Yuyan-C can you make a short summary of the workflow you/we proposed before the dev meeting next tuesday? I know you have a more detailed one in the works.

**Comment by mihow** [comment]
1) Decide on classification model for Panama species - 2 new options with better OOD results and expanded checklist. Likely the new model with ~2,360 species (Resnet50). Called "Panama Plus Species Classifier - Mar 2025"
  - TODO:
  - - Add classifier to AMI Data Companion (@Yuyan-C)
  - - Add new output field to API response for additional score(s) 
2) New column / field for "OOD Score" (either calculated from logits or existing softmax score). (Current score in Antenna is Softmax Score). Can score names in Antenna be dynamic? We can hardcode for now, but each model will likely want to add a definition of it's calibrated confidence score.
3) If occurrences OOD Score below threshold - Unknown Species (catch-all)
4) Decide on / Test 2 approaches. Start with vector database locally, start with pgvector query of current ADC database
5) Cluster all "Unknown Species" to Unknown Species n01, n02, etc. (based on feature vector of image classifier)
